comps/comps-katello-pulp-server-rhel6.xml

```diff
@@ -23,10 +23,6 @@
        <packagereq type="default">pulp-consumer-client</packagereq>
        <packagereq type="default">pulp-docker-plugins</packagereq>
        <packagereq type="default">pulp-docker-admin-extensions</packagereq>
-       <packagereq type="default">pulp-nodes-admin-extensions</packagereq>
-       <packagereq type="default">pulp-nodes-child</packagereq>
```

Comment:
I don't honestly know as we don't install a full (non-child) pulp currently today iirc